muzzle/src/main/java/io/opentelemetry/javaagent/tooling/HelperInjector.java

```diff
@@ -385,10 +389,23 @@ private static class HelperClassInjector {
     }
 
     Class<?> inject(ClassLoader classLoader, String className) {
+      // if security manager is present byte buddy calls
+      // checkPermission(new ReflectPermission("suppressAccessChecks"))
```

Comment:
I didn't understand this comment?

Reply:
I added the following to this comment `so we must call class injection with AccessController.doPrivileged when security manager is enabled`. Does it make sense now?